### ISSUE TITLE
Set default search engines on non-esr version?

### ISSUE THREAD

**Issue by Kreyren** [mozilla/policy-templates#1132]

**Comment by boredsquirrel** [comment]
duplicate of https://github.com/mozilla/policy-templates/issues/484

but I also highly support this

why restrict only this setting to ESR?

There are companies that don't use ESR, as ESR can have compatibility problems with some websites

**Comment by mkaply** [comment]
There was a concern about companies using it to hijack search. I'll work with the team to revisit that decision.

**Comment by Kreyren** [comment]
For context i am librehoster and administer the NiXium infrastructure at: https://github.com/Arcanyx-org/NiXium

The configuration is used to provide privacy-oriented settings by default to my users: https://github.com/Arcanyx-org/NiXium/blob/central/src/nixos/users/modules/web-browsers/firefox/firefox.nix

e.g. this is how i use it for my declarative firefox config: https://github.com/Arcanyx-org/NiXium/blob/central/src/nixos/users/users/kreyren/home/modules/web-browsers/firefox/firefox.nix

**Comment by celenityy** [comment]
Agreed, this would be very nice to have. We already support installing extensions, which is far more dangerous...

**Comment by Henry-Hiles** [comment]
Hi, if you're on nix you can use https://hedgedoc.grimmauld.de/s/rVnTq0-Rs#

**Comment by mkaply** [comment]
We're going to do this:

https://bugzilla.mozilla.org/show_bug.cgi?id=1961839
